cmd/root.go

```diff
@@ -67,7 +67,7 @@ func newRootCmd() *cobra.Command {
 	logLevelUsage := fmt.Sprintf("Set the logging detail level. Valid levels are: %v", log.GetLogLevels())
 	res.PersistentFlags().StringP("log-level", "l", "info", logLevelUsage)
 	res.PersistentFlags().BoolP("version", "V", false, "Version for asconfig.")
-	res.PersistentFlags().StringP("format", "F", "", "The format of the source file(s). Valid options are: yaml, yml, and conf.")
+	res.PersistentFlags().StringP("format", "F", "conf", "The format of the source file(s). Valid options are: yaml, yml, and conf.")
```

Comment:
How do you tell if they provided this flag or not? If their files do not provide extensions, do we just assume the file is a `.conf`?

Reply:
I believe `conf` will be displayed as the default in the help output. After reading getConfFileFormat I am still not sure why the default should be `conf`

Reply:
I asked @hev about the default. He thinks that .conf format files will be more common at first and wants to be able to paste them to stdin without using the format flag.

I was thinking that using .yaml as the default might be better since we want to move configs in that direction. It will make working with configs in yaml easier if they are the default format. Raising this again for Adam to confirm.

Reply:
At least for me `pbpaste` is one of the main ways I like to use pipes so I am sticking to my .conf default preference.

Reply:
made .conf the default